tests/PHPUnit/Integration/CronArchiveTest.php

```diff
@@ -931,6 +931,42 @@ private function getInvalidationsInTable($includeInvalidated = false)
         $sql = "SELECT idarchive, idsite, period, date1, date2, name, report$suffix FROM `$table`";
         return Db::fetchAll($sql);
     }
+
+    private function insertArchiveData($archivesToInsert)
```

Comment:
I'm just wondering where are we using these 3 methods?

Reply:
these were for some tests in a previous commit i removed, i'll remove these before it gets merged